### PR TITLE
Marcosertoli/equilibrium not nan

### PR DESCRIPTION
**Pull request by marcosertoli**

### CODE REVIEW

**Comment by marcosertoli** [comment]
* Resolved equilibrium issues due to R_shift - restricting (R,z) to where psi != NaN
* PPTS consistency with Indica native coordinates - rpos --> R, zpos --> z
* Working on fixing ASTRA...
* get_revision in mds_utils and in readers so that we can work with trees that don't have BEST.

File: indica/readers/st40reader.py [hunk 1]
```diff
@@ -269,26 +280,34 @@ def _get_helike_spectroscopy(
...
 
-        results["t"], _ = self._get_signal(uid, instrument, ":time", revision)
-        wavelength, _ = self._get_signal(uid, instrument, ":wavelen", revision)
+        results["t"], _ = self._get_signal(
+            uid, instrument, ":time", results["revision"]
+        )
+        wavelength, _ = self._get_signal(
+            uid, instrument, ":wavelength", results["revision"]
+        )
+        # TODO: change once wavelength in MDS+ has been fixed to nanometers!
```
Comment (michael-gemmell) [inline indica/readers/st40reader.py:301]: Why has this been readded? New XRCS runs don't need this.

**Review by michael-gemmell — CHANGES_REQUESTED** [review]
Looks good to me.

File: indica/readers/st40reader.py (hunk 1 above)
Comment (marcosertoli) [reply indica/readers/st40reader.py:301]: Error: will check it out and fix. Send it back you you by EOP today

**Review by michael-gemmell — APPROVED** [review]

**Pull request merged**